python/sglang/srt/mem_cache/radix_cache.py

```diff
@@ -278,8 +297,15 @@ def insert(self, key: RadixKey, value=None, chunked=False):
         if self.disable:
             return 0
 
+        key.token_ids = self.key_convert_fn(key.token_ids)
+
         if value is None:
             value = torch.tensor(key.token_ids, dtype=torch.int64)
+
+        if self.is_eagle:
```

Comment:
hiradix (and other trees like swa) override the `insert` function, would that be a problem since eagle worker shared the same tree?